src/client/common/extensions.ts

```diff
@@ -108,6 +112,17 @@ Promise.prototype.ignoreErrors = function <T>(this: Promise<T>) {
     this.catch(() => {});
 };
 
+/**
+ * Logs an error for any promise.
+ */
+Promise.prototype.logErrors = function <T>(this: Promise<T>) {
+    // eslint-disable-next-line no-empty, @typescript-eslint/no-empty-function
+    this.catch((e) => {
```

Comment:
I would avoid this, creating this means you MUST import `extesnsions.ts` everywhere, and the moment you miss it, things could fall over.